feat:Add diamond.js & equilateral.js(#39515)

nxfareed opened the pull request.

Commit:
feat: diamond.js & equilateral.js add

Prabal555 commented:
Please read the task clearly you are missing implementation of lots of instruction.

Commit:
feat:add package.json, permission.js, validation.js

nxfareed commented:
Added the package.json file for the task of Day 3:
Link to task of Day 3:  https://redmine.successive.tech/issues/39515

Also added the files for Day 4:
link for the task of Day 4:  https://redmine.successive.tech/issues/39517
files 'validation.js' and 'permissions.js' added to extras/utils
